pandora-client-web/src/components/wiki/pages/items.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react';
+
+export function WikiItems(): ReactElement {
+	return (
+		<>
+			<h2>Items</h2>
+
+			<h3>Introduction</h3>
+
+			<p>
+				Text block 1<br />
+				Text block 2
+			</p>
+
+			<h3>Item-specific features</h3>
+			<ul>
+				{
+					// <li><a href='#IT_'></a></li>
+				}
+				<li><a href='#IT_Room-level_items'>Room-level items</a></li>
+				<li><a href='#IT_Item_preferences_and_limits'>Item preferences and limits</a></li>
+			</ul>
+
+			<h4 id='IT_'>Feature name</h4>
+			<p>
+				Text block 1<br />
+				Text block 2
+			</p>
+			<ul>
+				<li>Subfeature 1</li>
+				<li>Subfeature 2</li>
+				<li>Subfeature 3</li>
+			</ul>
+
+			<h4 id='IT_Item_preferences_and_limits'>Item preferences and limits</h4>
+			<p>
+				You can set set preferences for individual items or groups of similar items or limit them with the "Item Limits"-tab in the wardrobe.
```

Comment:
Well, for me fav, normal and maybe are preferences, but prevent and no-render are limits, rather than preferences, therefore the "or" made sense for me.

But I understand how this is not obvious and may be confusing, so I will avoid this terminology issue by writing instead:
`With the "Item Limits"-tab in the wardrobe, you can set preferences for individual items or groups of similar items, such as limiting them.`